docs/src/prototypes/CopyToClipboard/index.tsx

```diff
@@ -27,27 +26,51 @@ const CopyToClipboardPrototype: React.FC<CopyToClipboardPrototypeProps> = props
 }
 
 const CopyToClipboardInMenu: React.FC = props => {
+  const item = {
+    key: 'edit',
+    content: 'Edit',
+    menu: [
+      'Open File...',
+      'Save File...',
+      render =>
+        render('Copy text', (Component, props) => {
+          return <CopyToClipboard value={'Julius Caesar'} trigger={<Component {...props} />} />
+        }),
+    ],
+  }
+
+  return <Menu items={[item]} />
+}
+
+const CopyToClipboardAttached: React.FC = props => {
+  const targetRef = React.useRef<HTMLElement>(null)
+
+  const item = {
+    key: 'edit',
+    content: 'Edit',
+    menu: [
+      'Open File...',
+      'Save File...',
+      render =>
+        render('Copy text', (Component, props) => {
+          return (
+            <CopyToClipboard
+              attach={{ target: targetRef.current }}
```

Comment:
```suggestion
              attach={{ target }}
```

https://github.com/stardust-ui/react/pull/1900/files#r322141672